src/effects/VST/VSTEffect.cpp

```diff
@@ -1758,24 +1758,11 @@ bool VSTEffectWrapper::Load()
    return success;
 }
 
-void VSTEffect::Unload()
-{
-   if (mAEffect)
-   {
-      // Finally, close the plugin
-      callDispatcher(effClose, 0, 0, NULL, 0.0);
-      mAEffect = NULL;
-   }
-}
```

Comment:
I think the correct thing to do in the "Unify Unload" commit is call Unload form the destructor of VSTEffectWrapper and remove the calls in the other destructors.

But note that this causes it to happen for the Validator too, which is also a wrapper.

As this commit stands, a Validator would have a Load at construction time, not paired with an Unload, and that might mean that you are leaking resources.

Reply:
Good point! done.